plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -253,26 +232,19 @@ private static void createPomXml(IProject project, AppEngineProjectConfig config
         "1.3.2"); //$NON-NLS-1$
     properties.put("mavenPluginVersion", mavenPluginVersion); //$NON-NLS-1$
 
-    String sdkVersion = getCurrentVersion(
-        "com.google.appengine", //$NON-NLS-1$
-        "appengine-api-1.0-sdk", //$NON-NLS-1$
-        "1.9.64"); //$NON-NLS-1$
-    properties.put("appEngineApiSdkVersion", sdkVersion); //$NON-NLS-1$
-    
+    properties.put("servletVersion", "3.1");
+    properties.put("compilerVersion", "1.8");
+
     if (isStandardProject) {
-      if (isStandardJava7RuntimeSelected(config)) {
-        properties.put("servletVersion", "2.5"); //$NON-NLS-1$ //$NON-NLS-2$
-        properties.put("compilerVersion", "1.7"); //$NON-NLS-1$ //$NON-NLS-2$
-      } else {
-        properties.put("servletVersion", "3.1"); //$NON-NLS-1$ //$NON-NLS-2$
-        properties.put("compilerVersion", "1.8"); //$NON-NLS-1$ //$NON-NLS-2$
-      }
-      createChildFile("pom.xml", Templates.POM_XML_STANDARD_TEMPLATE, //$NON-NLS-1$
-          project, properties, monitor);
-    } else {
-      createChildFile("pom.xml", Templates.POM_XML_FLEX_TEMPLATE, //$NON-NLS-1$
-          project, properties, monitor);
+      String sdkVersion = getCurrentVersion(
+          "com.google.appengine", //$NON-NLS-1$
+          "appengine-api-1.0-sdk", //$NON-NLS-1$
+          "1.9.64"); //$NON-NLS-1$
```

Comment:
not for this PR but we do probably need to bump this to 1.9.70

Reply:
Done in this PR.